libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/EmbedInsightDialogBase/EmbedInsightDialogBase.tsx

```diff
@@ -26,6 +26,7 @@ export type IEmbedInsightDialogBaseProps = {
     onCopyCode: (code: string, type: CopyCodeOriginType, codeType: EmbedType) => void;
     onOptionsChange: (opt: IReactOptions | IWebComponentsOptions) => void;
     onTabChange: (selectedTab: EmbedType) => void;
+    showWebComponentsTab?: boolean;
```

Comment:
Please move this prop to line 23, as props which are non-function should be next to each other, for the sake of consistency.